js/private/js_run_binary.bzl

```diff
@@ -107,6 +109,19 @@ def js_run_binary(
 
             This makes node binaries match the expected bazel paradigm.
 
+        run_from_execroot: Runs the `tool` entry_point out of the execroot output tree instead of the runfiles of `tool`.
```

Comment:
nit "run out of" is vernacular, I think you mean setting the working directory?

Reply:
Actually, no. The working directory remains the same. This changes which copy of the entry point we're calling. Node itself is still run out of runfiles (since that is where the platform specific node lives) but the entry point passing to node is now the one in the execroot output tree instead of the one in runfiles.